README.md

```diff
@@ -43,12 +43,22 @@ Here are some of the tasks available in the `justfile`:
 - `publish {{chain-id}}`: Publish the App to a network.
 - `wasm`: Optimize the contract.
 - `schema`: Generate the json schemas for the contract
-- `ts-codegen`: Generate the typescript client code for the contract
-- `ts-publish`: Publish the typescript client code to npm
+<!-- - `ts-codegen`: Generate the typescript client code for the contract -->
+<!-- - `ts-publish`: Publish the typescript client code to npm -->
 - `publish-schemas`: Publish the schemas by creating a PR on the Abstract [schemas](https://github.com/AbstractSDK/schemas) repository.
 
 You can see the full list of tasks available by running `just --list`.
 
+### Compiling
+
+Best to run `cargo update` to have synced versions just in case.
+
+You can compile your module by running the following command:
+```sh
+just wasm
+```
+This should result in an artifacts directory being created in your project root. Inside you will find a `my_module.wasm` file that is your module’s binary. Make sure it's the same name as mentioned in `Cargo.toml` file. If not, just edit it to match.
```

Comment:
We shouldn't have to match it, given https://github.com/AbstractSDK/cw-orchestrator/pull/334

Reply:
So let's hop on a call to debug if this is really needed.